core/container/src/main/java/org/wildfly/swarm/container/runtime/RuntimeDeployer.java

```diff
@@ -232,8 +234,18 @@ public void deploy(Archive<?> deployment, String asName) throws DeploymentExcept
                 }
             }
 
-            if (BootstrapProperties.flagIsSet(SwarmProperties.EXPORT_DEPLOYMENT)) {
-                final File out = new File(deployment.getName());
+            //if (BootstrapProperties.flagIsSet(SwarmProperties.EXPORT_DEPLOYMENT)) {
+            String exportLocation = System.getProperty(SwarmProperties.EXPORT_DEPLOYMENT);
```

Comment:
I'm thinking if maybe someone sets this property to `false` (maybe by some automation), but that's probably too much of a corner case.